lib/ansible/plugins/lookup/aws_account_attribute.py

```diff
@@ -0,0 +1,160 @@
+# (c) 2017 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = """
+lookup: aws_account_attribute
+author:
+  - Sloane Hertel <shertel@redhat.com>
+version_added: "2.5"
+requirements:
+  - boto3
+  - botocore
+short_description: Look up AWS account attributes.
+description:
+  - Describes attributes of your AWS account. You can specify one of the listed
+    attribute choices or omit it to see all attributes.
+options:
+  attribute:
+    description: The attribute for which to get the value(s).
+    default: null
+    choices:
+      - supported-platforms
+      - default-vpc
+      - max-instances
+      - vpc-max-security-groups-per-interface
+      - max-elastic-ips
+      - vpc-max-elastic-ips
+      - has-ec2-classic
+  boto_profile:
+    description: The boto profile to use to create the connection.
+    default: null
+    env:
+      - name: AWS_PROFILE
+      - name: AWS_DEFAULT_PROFILE
+  aws_access_key:
+    description: The AWS access key to use.
+    default: null
+    env:
+      - name: AWS_ACCESS_KEY_ID
+      - name: AWS_ACCESS_KEY
+      - name: EC2_ACCESS_KEY
+  aws_secret_key:
+    description: The AWS secret key that corresponds to the access key.
+    default: null
+    env:
+      - name: AWS_SECRET_ACCESS_KEY
+      - name: AWS_SECRET_KEY
+      - name: EC2_SECRET_KEY
+  aws_security_token:
+    description: The AWS security token if using temporary access and secret keys.
+    default: null
+    env:
+      - name: AWS_SECURITY_TOKEN
+      - name: AWS_SESSION_TOKEN
+      - name: EC2_SECURITY_TOKEN
+  region:
+    description: The region for which to create the connection.
+    default: null
+    env:
+      - name: AWS_REGION
+      - name: EC2_REGION
+"""
+
+EXAMPLES = """
+vars:
+  has_ec2_classic: "{{ lookup('aws_account_attribute', attribute='has-ec2-classic') }}"
+  # true | false
+
+  default_vpc_id: "{{ lookup('aws_account_attribute', attribute='default-vpc') }}"
+  # vpc-xxxxxxxx | none
+
+  account_details: "{{ lookup('aws_account_attribute', wantlist='true') }}"
+  # [{'AttributeName': 'supported-platforms', 'AttributeValues': [{'AttributeValue': 'VPC'}],...}]
```

Comment:
Do we want to flatten this in a nice way? Like `{'supported-platforms': ['VPC', ....]}` and such, similar to the tag-flattening we do?